.github/workflows/scheduled-prod-deploy.yml

```diff
@@ -0,0 +1,52 @@
+name: Scheduled Prod Deploy
+
+on:
+  schedule:
+    # Weekly Mondays & Thursdays @ 15:00 UTC (8am PST)
+    - cron: '0 15 * * 1,4'
+
+env:
+  DOCKER_BUILDKIT: 1
+  COMPOSE_DOCKER_CLI_BUILD: 1
+  DOCKER_REPO: ${{ secrets.ECR_REPO }}/
+  SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+
+jobs:
+  upgrade:
+    runs-on: ubuntu-20.04
+    environment:
+      name: prod
+    steps:
+      - uses: act10ns/slack@v1
+        with:
+          status: starting
+        if: always()
+      - name: Configure AWS Prod Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
+          role-to-assume: ${{ secrets.AWS_PROD_ROLE_TO_ASSUME }}
+          role-duration-seconds: 900
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+        run: git checkout HEAD^
+      - name: Install happy dependencies
+        run: |
+          pip install -r .happy/requirements.txt
+      - name: Update deployment
+        env:
+          TFE_TOKEN: ${{ secrets.TFE_TOKEN }}
+        run: |
+          ./scripts/happy --profile="" --env prod update prodstack --tag HEAD
```

Comment:
the --tag argument has to refer to a valid docker image tag. We push images to our ECR repos on every successful commit to trunk with the tags "branch-trunk" and "sha-${GITHUB_SHA:0:8}" ... the scripts/happy_deploy.py script basically looks for the latest successful build & push. We might want invoke that instead of the happy CLI directly here.

Reply:
The happy deploy script would launch another github action, and then this action would only know if the script successfully ran and not if the deployment was successful, right? Could I just pass in the tag "branch-trunk" or `sha-$(git rev-parse --short HEAD)`?

I tried passing HEAD on the cli and it worked so I thought it would work in the script.

Reply:
ahhh ... good point about the deploy script -- maybe we can it to output a valid tag instead of invoking the action, if we pass in a flag?

Reply:
that's a good idea - i'll expose the functionality of `get_latest_successful_deployment()`.